tutorials/time-freq/plot_ssvep.py

```diff
@@ -0,0 +1,786 @@
+"""
+=======================================
+Basic analysis of an SSVEP/vSSR dataset
+=======================================
+
+In this tutorial we compute the frequency spectrum and quantify signal-to-noise
+ratio (SNR) at a target frequency in EEG data recorded during fast periodic
+visual stimulation (FPVS).
+Extracting SNR at stimulation frequency is a simple way to quantify frequency
+tagged responses in MEEG (a.k.a. steady state visually evoked potentials,
+SSVEP, or visual steady-state responses, vSSR in the visual domain,
+or auditory steady-state responses, ASSR in the auditory domain).
+
+DATA:
+
+We use a simple example dataset with frequency tagged visual stimulation:
+N=2 participants observed checkerboards patterns inverting with a constant
+frequency of either 12Hz of 15Hz. 10 trials of 20s length each.
+32ch wet EEG was recorded.
+
+Data can be downloaded at https://osf.io/7ne6y/
+Data format: BrainVision .eeg/.vhdr/.vmrk files organized according to BIDS
+standard.
+
+OUTLINE:
```

Comment:
Instead of doing this, I would just make these three the second-level headings (the title of the example being the first-level heading):
```
Visualizing PSD and SNR spectrum
--------------------------------

Data preprocessing
^^^^^^^^^^^^^^^^^^

...

Estimating SNR at stimulation frequency
---------------------------------------

...

Statistically separating 12 and 15 Hz responses
-----------------------------------------------

...
```

Usually in RST you would then do a `.. contents:: local` or so, but it turns out we are about to update our website theme to automatically add this, e.g., the right bar here:

https://25174-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/machine-learning/plot_sensors_decoding.html

Reply:
this one is still open.. 
@larsoner I am not sure i entirely got your suggestion.. was it about reordering the sections, or about adding internal links, or both?

Reply:
The output currently looks like:

![Screenshot from 2021-02-24 12-57-27](https://user-images.githubusercontent.com/2365790/109044015-e6439b00-769f-11eb-93c4-7c46ff4108c3.png)

Notice that you have a custom/non-standard OUTLINE: section in the middle column, *and* the standard Sphinx-generated outline on the right, which is not the same. The sphinx one is automatically generated from the second-level headings in the file (the only first-level heading being the title of the example).

A better way to structure this is to make it so that whatever outline you want to show is the one automatically generated by Sphinx based on the first-level headings -- i.e., change your first level headings to have the content you want. And where you currently have `OUTLINE:` in your RST you can just put:
```
.. contents:: local
   :depth: 1
```
And it should make a nice little TOC for you. Having one inline is still useful even with the "right TOC" because in this theme below a specific width the right bar disappears:

![Screenshot from 2021-02-24 13-00-26](https://user-images.githubusercontent.com/2365790/109044347-4fc3a980-76a0-11eb-975b-f73581b37e56.png)

If this still isn't clear I can push a quick commit to do it

Reply:
adressed in 03ac0512c1344d72d0b2a7a77f187e1bb6e01187

hope this worked